package.json

```diff
@@ -35,5 +35,8 @@
   },
   "resolutions": {
     "@babel/core": "7.26.7"
+  },
+  "dependencies": {
+    "framer-motion": "^12.6.3"
```

Comment:
This should be added to the `website` package. Notice how all the dependencies in this file are the shared dev ones.